wiki/People/The_Team/Global_Moderation_Team/id.md

```diff
@@ -3,38 +3,43 @@ tags:
+
+ - masalah pada obrolan (cth. spam atau pelecehan)
+ - perilaku yang buruk di forum (cth. postingan yang tidak pantas)
+ - diskusi beatmap dan bagian komentarnya
+ - moderasi profil pengguna (NSFW atau konten yang tidak pantas pada avatar, sampul atau detail profil, ...)
 
-## Menjadi anggota Global Moderation Team
+## Menjadi seorang moderator
 
 Menjadi anggota dari Global Moderation Team membutuhkan jumlah kontribusi yang signifikan. Calon anggota sering diperhatikan dari kerelaan mereka untuk membantu pengguna lain, aktivitas di dalam komunitas, dan berkelakuan baik. Kemampuan mereka untuk berbicara bahasa kedua selain bahasa Inggris sangat disukai. Kandidat didiskusikan di dalam tim untuk menentukan penambahan potensial sebagai member tim.
 
 ### Bagaimana cara agar dipertimbangkan
 
-Faktor terpenting agar dipertimbangkan adalah perilaku, aktivitas, pemahaman terhadap peraturan, dan kesabaran:
+Faktor terpenting agar dipertimbangkan adalah perilaku, aktivitas, pemahaman terhadap [peraturan](/wiki/Rules), dan kesabaran:
 
 1. Demonstrasikan perilaku yang baik dengan bersikap sopan dan memperlakukan pengguna lain dengan hormat. Jaga akun kamu terbebas dari silence atau masalah serius yang sangat penting.
-2. Demonstrasikan aktivitas in-game dengan berpartisipasi di obrolan dan/atau forum. Laporkan perilaku buruk dan masalah melalui sistem [report](/wiki/Reporting_Bad_Behaviour). Jawab pertanyaan-pertanyaan dan bantu selesaikan isu yang dialami pengguna lain.
+2. Demonstrasikan aktivitas in-game dengan berpartisipasi di obrolan dan/atau forum. [Laporkan perilaku buruk](/wiki/Reporting_Bad_Behaviour) dan masalah melalui sistem report. Jawab pertanyaan-pertanyaan dan bantu selesaikan isu yang dialami pengguna lain.
 3. Memiliki pemahaman yang cukup tentang [peraturan](/wiki/Rules) serta penerapannya. Penting untuk mengetahui apa saja yang dapat diterima dan tidak diterima di osu!
 4. Bersabarlah. GMT tidak menambahkan secara spontan anggota yang telah membantu hanya beberapa hari. Butuh waktu bagi GMT untuk mengevaluasi seseorang⁠ — kepribadian mereka, apa yang dapat mereka bantu, aktivitas mereka, dll.
 
-Umumnya, tim akan mengamati orang selama beberapa minggu untuk memastikan bahwa mereka layak ditambahkan ke tim. Harap bersabar dan jangan berlebihan. Jangan ragu untuk bertanya kepada siapa pun dari GMT tentang pertanyaan apa pun terkait peraturan, melaporkan pengguna, dan masalah lainnya.
+Umumnya, tim akan mengamati orang selama beberapa minggu atau bulan untuk memastikan bahwa mereka layak ditambahkan ke tim. Harap bersabar dan jangan berlebihan. Jangan ragu untuk bertanya kepada siapa pun dari GMT tentang pertanyaan apa pun terkait peraturan, melaporkan pengguna, dan masalah lainnya.
```

Comment:
`orang` --> `kandidat-kandidat yang dainggap layak` maybe? 

`orang` feels... out of touch here :(